webpack.config.js

```diff
@@ -12,6 +13,33 @@ const isDev = ! isProd && ! isI18n;
 
 const browsers = 'last 2 versions, not ie_mob 10, not ie 10';
 
+const cssLoaders = [
+	isDev ? 'style-loader?hmr=false' : MiniCssExtractPlugin.loader,
+	'css-loader',
+	{
+		loader: 'postcss-loader',
+		options: {
+			plugins: () => [
+				autoprefixer( { browsers } ),
+				url( {
+					url: ( asset ) => 'https://wordpress.com/' + asset.url,
```

Comment:
`css-loader` doesn't have this functionality anymore, they recommend to use this `postcss` plugin instead.